apps/hashdotai/LICENSE.md

```diff
@@ -1,9 +1,9 @@
 # License
 
-The [hash.ai](https://hash.ai) website contains educational material and general information regarding HASH.
+**Copyright © 2019–, HASH. All Rights Reserved.**
 
-Markdown files (those suffixed `.md`) containing text displays on the public site at [hash.ai](https://hash.ai/) are available under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International](https://github.com/hashintel/hash/blob/main/.github/licenses/LICENSE-CC.md) license.
+The [hash.ai](https://hash.ai) website contains educational material and general information regarding HASH.
 
-All other files are released under the [GNU Affero General Public License 3.0](https://github.com/hashintel/hash/blob/main/.github/licenses/LICENSE-AGPL.md), or as otherwise disclaimed in line with our root repository [LICENSE](https://github.com/hashintel/hash/blob/main/LICENSE.md).
+Unlike the vast majority of this repository, the contents of this particular directory are not open-source, but have nevertheless been made available on GitHub so that their provenance and integrity can be confirmed.
```

Comment:
As this is pretty much source available and not open source (which is totally okay btw!) have you considered licenses such as:

* https://creativecommons.org/licenses/by-nc-nd/4.0/
* https://www.gnu.org/licenses/fdl-1.3.html

Reply:
In the current directory the:
- **code** is open-source (AGPL), although there is little of it;
- **the** content is available under Attribution-NonCommercial-ShareAlike 4.0 International.

However, the directory's contents are going to be substituted _in whole_ in the upcoming period, and replaced with a new public-facing website (which we do not intend to license externally at this time as we explicitly want the brand's marketing website look, feel and form to remain unique).

We'll be keeping the `docs` and `glossary` directory contents freely licensed under their current CC arrangements, however, which I've reflected now (to prevent any disruption in availability) in this PR through https://github.com/hashintel/hash/commit/5ab6ae118954f30f5b1eb52180e946300488ed77 and https://github.com/hashintel/hash/commit/839d2388d758df54f9514d0f7fadeb8f478d5ef7 -- so thank you for the prompt! n.b. I believe the second of the two commits might result in issues rendering on the current [hash.ai/glossary](https://hash.ai/glossary) site, so we'll need to confirm that and handle it before merging this now.